plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -369,8 +371,14 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       .update({
         next_update_at:
           tx.client.config.client === 'sqlite3'
-            ? tx.raw(`datetime('now', ?)`, [`100 seconds`])
-            : tx.raw(`now() + interval '100 seconds'`),
+            ? tx.raw(`datetime('now', ?)`, [
+                `${this.options.refreshIntervalSeconds} seconds`,
```

Comment:
Prolly cast this one to a number too, just cause

Reply:
In a throwing way then right? Otherwise the value of doing it is limited. It's at least SQL safe since it's a parameterised query. Best case we'd pass the string NaN or whatnot and have it fail in the SQL layer otherwise